frontend/src/slices/chats/actions.ts

```diff
@@ -59,4 +60,29 @@ const createMessage = createAsyncThunk<
   return await chatMessagesApi.createChatMessage(payload);
 });
 
-export { createChat, createMessage, getAllChats, getCurrentChatMessages };
+const deleteChat = createAsyncThunk<
+  ChatGetAllItemResponseDto[],
+  number,
+  AsyncThunkConfig
+>(`${sliceName}/delete-chat`, async (id, { extra, getState, dispatch }) => {
+  const { chatApi } = extra;
+  await chatApi.deleteChat(id);
+
+  const {
+    chats: { chats },
+  } = getState();
+
+  dispatch(appActions.navigate(AppRoute.CHATS));
+
+  return chats.filter((chat) => {
```

Comment:
You can do it in the slice in case of `deleteChat.fulfilled`. At least it's approach we already get used to and also you wouldn't use exta stuff, like `getState`

Reply:
I am not sure that it is a good way to do it in the slice. Slice must be as simple as it can be. Also, I need to pass deletedId to slice to make this logic there, so it looks weird to me.

Reply:
Actually I would pass the id in similar situation, but it's quite controversial question

Reply:
So maybe @v0ldemar01 or @fshabanov can clarify for me, where would it be better to place this logic: in action or slice?

Reply:
It is ok to stay with current approach

Reply:
Ok, thanks!